docs/section11/Rule_11-1.md

```diff
@@ -0,0 +1,63 @@
+**Data Lookup:**   
+**Function Call:**  
+- **get_component_by_id**  
+- **compare_swh_dist_systems_and_components**
+- **compare_context_pair** - there is no RDS for this function, but it is a function developed for Rule 1-6 that compares two elements  
+- **get_swh_components_associated_with_each_swh_distribution_system**  
+
+
+**Applicability Checks:**
+- check that the U_RMD has SWH Uses with loads
+
+## Applicability Checks:
+- in this case, all of the SWH Distribution IDs identified when the custom context was created are applicable to this rule.
+- Look at each swh distribution system id: `for swh_dist_system_id in context: CONTINUE TO RULE LOGIC`
+
+  ## Rule Logic #
+  - there are a couple cases here:
+    1.  If the swh distribution exists in the user model, AND has swh use, then expect user and proposed systems to be match
+    2.  Otherwise, expect the distribution system to match between proposed and baseline
+    3.  Systems that don't do 1 or 2 fail
+  
+  - create a variable user_proposed_str, which is the string lookup used by compare_context_pair for comparing elements between the user and proposed RMDS: `user_proposed_str = "AppG 11-1 P_RMD Equals U_RMD"`
+  - create a variable base_proposed_str, which is the string lookup used by compare_context_pair for comparing elements between the user and proposed RMDS: `base_proposed_str = "AppG 11-1 P_RMD Equals B_RMD"`
+  - set a list user_proposed_comparison to [].  If the end of the rule is reached and this list (and the corresponding baseline list) still have zero length, then the rule passes.  Otherwise the list will include a series of messages for the user about why it didn't pass: `user_proposed_comparison = []`
+  - set a list user_baseline_comparison to [].  If the end of the rule is reached and this list (and the corresponding proposed list) still have zero length, then the rule passes.  Otherwise the list will include a series of messages for the user about why it didn't pass: `user_baseline_comparison = []`
+  
+  - Check whether the system exists in the user model: `if u_swh_system_and_equip_dict[swh_dist_system_id]:`
+    - check whether the system has SWH uses in the user model: `if u_swh_system_and_equip_dict[swh_dist_system_id]["USES"]:`
+      - get the user swh_use: `u_swh_use = get_component_by_id(U_RMD, swh_use_id)`
+      - if the u_swh_use exists, and is greater than zero: `if u_swh_use != NULL and u_swh_use.use > 0:`
+        - compare the proposed and user models using the function compare_swh_dist_systems_and_components.  This function returns a list of errors.  If the list of errors has a length of 0, then the comparison encountered no issues: `user_proposed_comparison = compare_swh_dist_systems_and_components(P_RMD,U_RMD,user_proposed_str,swh_dist_system_id)`
+        - from here, go directly to the rule assertion: `GO TO RULE ASSERTION`
```

Comment:
This is implying that a rule outcome is given for each SWH Distribution System. Is that correct?

Reply:
for each SWH Distribution system ID, yes.